app/mailers/katello/subscription_mailer.rb

```diff
@@ -1,10 +1,13 @@
 module Katello
   class SubscriptionMailer < ApplicationMailer
+    helper :'katello/subscription_mailer'
     after_action :prevent_sending_blank_report
+    include SubscriptionMailerHelper
 
     def subscription_expiry(options)
       user = ::User.find(options[:user])
       days_from_now = options[:query]
+      start_report_task(days_from_now)
```

Comment:
Last comment from me - just need to ensure we don't start the report task if we're not going to send the email. I suppose we could do it at the bottom of this method `start_report_task(days_from_now) unless @pools.empty?`